config/default.conf

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
Does this redirect all traffic to index.html? That’s what I recommend.

Reply:
Exactly that.  The `nginx.conf` serves the root we tell it.  that server block allows us to do a lot of fun web server stuff, redirects, exposures, and ultimately reverse proxying.